### DISCUSSION TITLE
How do i upload Folders with files from Android

### DISCUSSION THREAD

**Discussion by XtarsAgency** [community/community#61032] [New to GitHub] [answered]
Hello, so im making a project and i want to import a folder to the repository. Wierdly, in android i can only upload selected Files but not selected Folders. Folders cannot be selected by my phone (Xiaomi 6A)

Can you maybe help? Because "Drag Files here" doesnt work

**Comment by nirav-gajera** [comment, marked as answer]
1. Create a compressed (zip) file containing the folder and its contents that you want to import to the repository.
2. On your Android device, navigate to the GitHub repository where you want to import the folder.
3. Tap on the "Add file" button or the "+" icon to add a new file.
4. Instead of selecting individual files, choose the option to upload a file from your device's storage.
5. Locate and select the compressed (zip) file that contains the folder you want to import.
6. GitHub will upload the compressed file to the repository.
7. Once the file is uploaded, you can open it within the repository and extract its contents to the desired location.

-By compressing the folder into a zip file, you can upload it as a single file to the repository. After uploading, you can extract the contents of the zip file within the repository to achieve the same result as uploading the folder directly

**Comment by techguy16** [comment]
You need to use the web interface to upload files on Android.

**Comment by Rohan-San** [comment]
I'd faced this issue initially too. A primitive solution that works perfectly is:

1. Open GitHub on your web browser (not the app)
2. Click on the 3 dots on the top right of the screen and enable "desktop site"
3. Go to your repository and click on add files.
4. Write the folder name and add '/' after it to create a folder. Then you can upload the files in this folder

![Screenshot_20230717_122044_Chrome](https://github.com/community/community/assets/107957865/32101058-6275-4634-8d6f-562df093d10b)
![Screenshot_20230717_122156_Chrome](https://github.com/community/community/assets/107957865/f9b423c1-22f6-41a6-aad6-af2182c97b96)

Hope this helped!

**Reply by techguy16** [reply]
That's what I said. It's hard but it works.

**Reply by Randomgituser69** [reply]
I can't see the miscellaneous option anywhere

**Reply by Chukwuemeka119** [reply]
Not working

**Reply by Amin0x** [reply]
Not work in android you can't upload folder like that

**Comment by XtarsAgency** [comment]
Thank you everyone, now i can upload folders now!

**Reply by Cryptoefx** [reply]
What's the solution ? Tell us !

**Reply by techguy16** [reply]
> What's the solution ? Tell us !

Use GitHub Codespaces for the repository, where you are able to upload the folder from your phone/tablet. The UI is quite small, but you are able to zoom in and see the icons.
